src/Util/MapUtil/MapUtil.js

```diff
@@ -244,6 +244,23 @@ export class MapUtil {
     return layerCandidates;
   }
 
+  /**
+   * Returns the list of layers matching the given pair of properties.
+   *
+   * @param {ol.Map} map The map to use for lookup.
+   * @param {String} key The property key.
+   * @param {Object} value The property value.
+   *
+   * @return {ol.layer.Base} The list of matching layers.
```

Comment:
`{ol.layer.Base}` -> `{ol.layer.Base[]}`

Reply:
Fixed in 52896b2ce70e